scraper/scraper/pipelines.py

```diff
@@ -0,0 +1,18 @@
+from scrapy.utils.project import get_project_settings
+import csv
+
+
+class SaveMepsCSV(object):
+    meps = []
+
+    def process_item(self, item, spider):
+        if spider.name == 'mep_xml':
```

Comment:
If you only want this to run when `spider.name` == `mep_xml`, then you can do:
```python
if spider.name != 'mep_xml':
    return
```
and remove one level of indentation.

Same idea applies for next function.